package.json

```diff
@@ -100,5 +100,9 @@
       "admin/",
       "packages/"
     ]
+  },
+  "dependencies": {
+    "ollama": "^0.5.1",
+    "passport-ldapauth": "^3.0.1"
```

Comment:
this needs to be removed from the /package.json file. please move the dependencies to the right folder if needed.